app.py

```diff
@@ -1,27 +1,63 @@
-from flask import Flask, request
 from database import db
-import user
```

Comment:
in line 3 we import classes directly.
in file models/__init__.py you can appreciate that we make all imports as a module. So in models we can access to classes